bodhi/tests/server/services/test_updates.py

```diff
@@ -3644,7 +3644,7 @@ def test_autopush_non_critical_update_with_no_negative_karma(self, publish, *arg
         self.assertEquals(up.autokarma, True)
 
         up = self.db.query(Update).filter_by(title=resp.json['title']).one()
-        self.assertEquals(up.request, UpdateRequest.stable)
+        self.assertEquals(up.request, UpdateRequest.batched)
 
     @mock.patch(**mock_valid_requirements)
     @mock.patch('bodhi.server.notifications.publish')
```

Comment:
Let's add another 2 tests to this module:

1. One that asserts that security updates skip batched and go straight to stable when getting the right karma.
2. One that asserts that high severity updates skip batched and go straight to stable when getting the right karma.

I did a quick search and didn't see existing tests like the above, but it's possible I missed them.